packages/squiggle-lang/src/public/SqValue/SqBoxed.ts

```diff
@@ -26,4 +26,12 @@ export class SqBoxed {
     const showAs = this.args.showAs();
     return showAs ? wrapValue(showAs, this.context) : undefined;
   }
+
+  numberFormat(): string | undefined {
+    return this.args.numberFormat();
+  }
+
+  dateFormat(): string | undefined {
+    return this.args.dateFormat();
+  }
```

Comment:
I imagine eventually we might want to have different boxed types. Like, ``SqDateBoxed``. I kind of want to wait until it's more needed now - at this point, it's not clear to me yet which direction we want to go.